### ISSUE TITLE
Emergent Elm - Arborist Nails

### ISSUE THREAD

**Issue by anonahmus** [EdenServer/community#5406]
### Checklist

- [ X] I have searched for existing issues for issues like this one. The issue has not been posted.
- [ ] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [X ] I have provided my client version in the details. (type /ver into your game window)
- [ ] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

Client version: 30181205_0

Per EDENWIKI , Arborist Nails should be 20-25% drop rate. I have successfully killed the NM about 10 times with TH4 and the feet have yet to drop. 

https://classicffxi.fandom.com/wiki/Emergent_Elm

**Comment by Mozzy4Ever** [comment]
Wiki drop rates are bad because they don't include TH levels. FFXIDB (http://www.ffxidb.com/zones/52/emergent-elm) doesn't have a good enough sample size on them sadly. Note that it has 63 kills and I still say it's not a good sample size.

**Comment by anonahmus** [comment]
> Wiki drop rates are bad because they don't include TH levels. FFXIDB (http://www.ffxidb.com/zones/52/emergent-elm) doesn't have a good enough sample size on them sadly. Note that it has 63 kills and I still say it's not a good sample size.

If that is the case, should EDENWIKI be updated to the correct percentage level that it is currently on? Because show 20-25% when that is not the drop rate in Eden would be misleading.

**Comment by bliven-was-taken** [comment]
Yep, and anyone can fix it, so gogo!

**Comment by anonahmus** [comment]
> Yep, and anyone can fix it, so gogo!

Aren't in-game drop rates manually adjusted by staff based on era accuracy? It seems since there are not enough source for the drop rate % then this appears to be a subjective case and if we have an idea of what the staff was basing their % level on I will be more than happy to edit :)

**Comment by maxdecphoenix** [comment]
i actually wrote a drops tracking addon to solve this exact problem.  while i'm sure the devs can look at the server metrics to see kill logs or item dispensations to verify things, players can't.  I got so pissed off killing tonberries for a rancor piece a couple weeks back that, after submitting 'Validation Ticket', that i wrote a passive items parser to log drops (including gil) to build an ffxidb.com style database specifically for eden.   You load it, play the game, it builds a log of every kill, and i use that log to build-up populate database with Eden specific information.  I've already found a couple anomolies with low-tier trashmobs supposedly dropping trash items don't actually drop those items on eden.  it functions well enough for me but a public release would need a bit o polishing and i've been working on other personal projects.
